pkg/volume/storageos/storageos_util.go

```diff
@@ -375,6 +376,10 @@ func getLoopDevice(path string, exec utilexec.Interface) (string, error) {
 	args := []string{"-j", path}
 	out, err := exec.Command(losetupPath, args...).CombinedOutput()
 	if err != nil {
+		if losetupInvalidOpt(out) {
+			return getLoopDeviceFromSysfs(path)
+		}
```

Comment:
Is getLoopDeviceFromSysfs supported everywhere? Would it be better to use instead `losetup` in all cases?

Reply:
@jsafrane - I think it can be substituted for just `-j` because `losetup` does something similar. In other cases like `-f --show`, for instance we'd have to call this function depending on the output.

(Also, thanks for taking the time to review this PR.)